tests/test_template_init.py

```diff
@@ -113,7 +113,12 @@ def init_git(path: Path):
 
 def run_command(command: str, cwd: Path, **kwargs) -> None:
     """Run a command, showing stdout/stderr on errors."""
-    default_kwargs = {"shell": True}
+    default_kwargs = {
```

Comment:
This helped me see that it can't run the hatch environment. hatch build --clean seems to work but not running the tests.